tests/core/mem-cgroup-disabled/task.yaml

```
@@ -11,6 +11,13 @@ prepare: |
     echo "Skipping non-grub device test"
     exit 0
   fi
+  # external backends would fail because a signed gadget cannot be replaced
+  # with an unsigned one
+  # TODO: remove once https://github.com/snapcore/snapd/pull/11589 landed
+  if ! snap list pc | grep -q " x[0-9]+ "; then  
```

Doesn't this require -E ?
```suggestion
  if ! snap list pc | grep -Eq " x[0-9]+ "; then  
```